kommunicate/src/main/java/io/kommunicate/zendesk/KmZendeskSendMessageTask.java

```diff
@@ -0,0 +1,61 @@
+package io.kommunicate.zendesk;
+
+import android.content.Context;
+import android.os.AsyncTask;
+
+import io.kommunicate.callbacks.KmCallback;
+import zendesk.chat.Attachment;
+
+public class KmZendeskSendMessageTask extends AsyncTask<Void, Void, KmZendeskApiModel> {
+    private Context context;
+    private String message;
+    private String displayName;
+    private String agentId;
+    private Integer conversationId;
+    private Long messageTimestamp;
+    //private boolean attachment;
+    private Attachment attachment;
+    private KmCallback kmCallback;
+
+
+    private KmZendeskSendMessageTask(Context context, String message, Attachment attachment, String displayName, String agentId, Integer conversationId, Long messageTimestamp, KmCallback kmCallback) {
+        this.context = context;
+        this.message = message;
+        this.displayName = displayName;
+        this.agentId = agentId.replace(":", "-");
```

Comment:
what is the purpose of this?

Reply:
Zendesk sends agentId in the format of "agent:11431678362897" but on our Kommunicate server we accept the request body as "agent-11431678362897"

Reply:
add the same in the code comment so that in future if any developer looks at it, they get the reason why its done this way

Reply:
Added comments to it